### PR TITLE
Grammar fixes to atom descriptions

### PR DESCRIPTION
**Pull request by SpaceManiac**
- Capitalization and punctuation on most descriptions
- CentCom instead of centcom where appropriate
- Earth instead of earth where appropriate

Descriptions without initial capitals and without ending punctuation have been fixed. Certain short descriptions or descriptions which are not supposed to show in game have not been modified.

Inconsistent uses of "centcom", "central command", "central", "centcomm", "cent. com.", &c. have been replaced with "CentCom", "Central Command", or "Central" as appropriate.

"Earth" has been capitalized when used to refer to the planet and uncapitalized when used as an adjective.

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@ update_label("John Doe", "Clowny")
 
 /obj/item/card/id/ert
 	name = "\improper CentCom ID"
-	desc = "A ERT ID card"
+	desc = "A ERT ID card."
```
Comment (ShizCalev) [inline code/game/objects/items/cards_ids.dm]: An ERT

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/miniantags/monkey/monkey.dm [hunk 2]
```diff
@@ -114,5 +114,5 @@
 		to_chat(world, "<span class='userdanger'>The staff managed to contain the monkey infestation!</span>")
 
 /datum/game_mode/monkey/generate_report()
-	return "Reports of an ancient [pick("retrovirus", "flesh eating bacteria", "disease", "magical curse blamed on viruses", "bananna blight")] outbreak that turn humans into monkies has been \
-			reported in your quadrant.  Any such infections may be treated with bananna juice.  If an outbreak occurs, ensure the station is quarantined to prevent a largescale outbreak at Centcom."
+	return "Reports of an ancient [pick("retrovirus", "flesh eating bacteria", "disease", "magical curse blamed on viruses", "banana blight")] outbreak that turn humans into monkeys has been \
+			reported in your quadrant.  Any such infections may be treated with banana juice.  If an outbreak occurs, ensure the station is quarantined to prevent a largescale outbreak at CentCom."
```
Comment (Cyberboss) [inline code/game/gamemodes/miniantags/monkey/monkey.dm]: Oneline this string please

File: code/game/objects/items/devices/radio/headset.dm [hunk 3]
```diff
@@ -84,7 +84,7 @@
 
 /obj/item/device/radio/headset/headset_eng
 	name = "engineering radio headset"
-	desc = "When the engineers wish to chat like girls. \nTo access the engineering channel, use :e. "
+	desc = "When the engineers wish to chat like girls. \nTo access the engineering channel, use :e."
```
Comment (Cyberboss) [inline code/game/objects/items/devices/radio/headset.dm]: Remove the space between . and \n

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by SpaceManiac** [comment]
For the onelining and space removal, do you want the other instances in those same files fixed?

**Comment by Cyberboss** [comment]
That'd be nice yeah

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**